bika/lims/browser/analysisrequest/publish.py

```diff
@@ -721,7 +721,18 @@ def __call__(self, ar, overwrite=False):
         if not overwrite:
             # Prevent any error related with digest
             data = ar.getDigest() if hasattr(ar, 'getDigest') else ''
-            if data:
+            # Check if the department managers have changed since the
+            # verification:
+            saved_managers = data.get('managers', {})
+            saved_managers_ids = set(saved_managers.get('ids', []))
+            current_managers = self.context.getManagers()
+            current_managers_ids = set([man.getId() for man in
+                                        current_managers])
+            # The symmetric difference of two sets A and B is the set of
+            # elements which are in either of the sets A or B but not in both.
+            are_differenet = saved_managers_ids.symmetric_difference(
+                current_managers_ids)
+            if data and len(are_differenet) == 0:
```

Comment:
I'd say this conditional ``if data`` should be placed in line 723, like this:
```python
if not overwrite 
    # Prevent any error related with digest
    data = ar.getDigest() if hasattr(ar, 'getDigest') else None
    if data:
        # Check if the department managers have changed since the
        # verification:
        saved_managers = data.get('managers', {})
        ....
        if len(are_different) == 0:
            ....
```
```